docs/learn/learn-polkadot-opengov.md

```diff
@@ -334,27 +333,47 @@ Once the proposal exits the Lead-in Period and enters the Voting Period, to be a
 satisfy the approval and support criteria for the **Confirmation Period**.
 
 - **Approval** is defined as the share of approval (_aye_ votes) vote-weight (after adjustment for
-  [conviction](#voluntary-locking)) against the total vote-weight (for all approval, rejection, and
-  abstained).
-- **Support** is the total number of votes (ignoring any adjustment for conviction) compared to the
-  total possible votes that could be made in the system.
-
-The figure shows that even if the approval criteria are satisfied (i.e. % approval is greater than
-the approval curve), the proposal only enters the confirmation period once the support criteria are
-satisfied (i.e. % support is greater than the support curve). If the referendum meets the criteria
-for the confirmation period, then the proposal is approved and scheduled for enactment. The
-Enactment Period can be specified when the referendum is proposed but is also subject to a minimum
-value based on the Track. More powerful Tracks enforce a larger Enactment Period to ensure the
-network has ample time to prepare for any changes the proposal may bring.
+  [conviction](#voluntary-locking)) against the total vote-weight (_aye_, _nay_, and _abstained_).
+- **Support** is the total number of _aye_ and _abstain_ votes (ignoring any adjustment for
+  conviction) compared to the total possible votes that could be made in the system. In case of
+  _split_ votes, only _aye_ and _abstain_ will count.
+
```

Comment:
```suggestion

:::info Nay votes are not counted towards Support

  Support is a measure of voters who turned out either in favor of the referenda and who consciously abstained from it. Support does not include _nay_ votes. This avoids edge situations where _nay_ votes could push a
  referendum into confirming state. For example, imagine current approval is high (near 100%, way
  above the approval curve), and current support is just below the support curve. A _nay_ could bump
  support above the support curve but not reduce approval below the approval curve. Therefore
  someone voting against a proposal would make it pass. Hence, a decrease in % of current approval through new 
 votes does not directly translate into increasing support because Support needs to consider _nay_ votes. 

:::

```